pkg/hub/controllers/slice_controller.go

```diff
@@ -122,6 +125,7 @@ type SliceReconciler struct {
 }
 
 var sliceFinalizer = "controller.kubeslice.io/hubSpokeSlice-finalizer"
+var slice_controllerName string = "workerslice_controller"
```

Comment:
variablename can't slice_controllerName

Reply:
are you suggesting "slice_controller_name" should be the variable name ?